modules/scrabble.js

```diff
@@ -110,7 +111,7 @@ module.exports = {
   events: {
     message: function (bot, nick, to, text) {
       if (text.match(bot.config.get('irc.controlChar') + 'scrabble')) return
-      let phrase = text.split(/[ '"]/)
+      let phrase = text.toUpperCase().split(/[ '"]/)
```

Comment:
I'm not sure we want to transform the string here.
It could be better to add a `normalisedWord` property to the object returned by the `computeWord()` function, and compare against `result.normalisedWord` instead of `result.word`.